exercises/dominoes/dominoes_test.rb

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'dominoes'
+
+# Test data version:
+# 82eb00d
+class DominoesTest < Minitest::Test
+  def test_empty_input_empty_output
+    # skip
+    input_dominoes = []
+    output_chain = Dominoes.chain(input_dominoes)
+    assert_correct_chain(input_dominoes, output_chain)
+  end
+
+  def test_singleton_input_singleton_output
+    skip
+    input_dominoes = [[1, 1]]
+    output_chain = Dominoes.chain(input_dominoes)
+    assert_correct_chain(input_dominoes, output_chain)
+  end
+
+  def test_singleton_that_can_not_be_chained
+    skip
+    input_dominoes = [[1, 2]]
+    output_chain = Dominoes.chain(input_dominoes)
+    assert_nil output_chain, "There should be no chain for #{input_dominoes}"
```

Comment:
What do you think about making a helper method `refute_correct_chain`?
I'm not entirely sure it's a good idea, but I like the symmetry of all the tests using similar terminology.